Frameworks/UIKit/StarboardXaml/StarboardXaml.cpp

```diff
@@ -95,14 +95,28 @@ void App::OnActivated(IActivatedEventArgs^ args) {
             _ApplicationMainLaunch(ActivationTypeToast, argsString);
         }
         UIApplicationMainHandleToastNotificationEvent(Strings::WideToNarrow(argsString->Data()).c_str());
+    } else if (args->Kind == ActivationKind::VoiceCommand) {
+        Windows::Media::SpeechRecognition::SpeechRecognitionResult^ argResult = safe_cast<VoiceCommandActivatedEventArgs^>(args)->Result;
+        TraceVerbose(TAG, L"Received voice command with argument - %s", argResult->Text->Data());
+        if (initiateAppLaunch) {
```

Comment:
nit: coding conventions state that a new if(){} block should be preceding by a new line.